app/users/page.tsx

```diff
@@ -1,59 +1,34 @@
-'use client';
-import React from 'react';
+'use client'
+import React, { useState } from 'react';
```

Comment:
I dont think you are using `useState` in this file so you can remove this import